views/showcase.html

```diff
@@ -339,6 +339,14 @@ <h2 id="readme" class="section-title">Read me</h2>
     var query = document.getElementById('search-input').value;
     window.location.href = '/search?q=' + query;
   }
+
+  function sendEvent(fileFormat, eventCategory, resourceTitle) {
+      gtag('event', fileFormat.toUpperCase() + " downloaded", {
```

Comment:
We're using the custom event - here we have nice docs - https://developers.google.com/analytics/devguides/collection/gtagjs/events.

You defined `action` as `FORMAT downloaded` which I don't think a good idea. It would be better to have a unified action name, e.g., `download` so that all file downloads are under the single action. Then you can play with event category, label and value so that we can differentiate datasets/resources and formats:

* `event_category`: dataset/package name
* `event_label`: resource name
* `value`: downloaded format

Reply:
This way, when I view the analytics reports, I can see how many `download` events happened in total. Then I click on that event and it will show me downloads by category (dataset). Next, I click on one of the category (dataset) so I can see downloads per resource within that dataset...and the same for format.